monkestation/code/modules/storytellers/converted_events/event_overrides.dm

```diff
@@ -230,23 +237,24 @@
 /datum/round_event_control/stray_meteor
 	track = EVENT_TRACK_MODERATE
 	tags = list(TAG_DESTRUCTIVE, TAG_SPACE, TAG_EXTERNAL)
-	shared_occurence_type = SHARED_METEORS
+	event_group = /datum/event_group/debris
 
 /datum/round_event_control/supermatter_surge
 	track = EVENT_TRACK_MODERATE
 	tags = list(TAG_DESTRUCTIVE, TAG_COMMUNAL)
-	shared_occurence_type = SHARED_BSOD
+	event_group = /datum/event_group/error
 
 /datum/round_event_control/tram_malfunction
 	track = EVENT_TRACK_MUNDANE
 	tags = list(TAG_COMMUNAL)
-	shared_occurence_type = SHARED_BSOD
+	event_group = /datum/event_group/error
```

Comment:
might want to shrink down the error event group

Reply:
error group has a somewhat short cooldown range anyways